src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -24,12 +24,26 @@ const determinePinIcon = (data, pinValue, pinnedRun) => {
   return null;
 };
 
+const determinePinDelta = (data, pinValue, pinnedRun) => {
+  if (
+    data.runId !== pinnedRun &&
+    typeof data.value === 'number' &&
+    data.value !== pinValue
+  ) {
+    const delta = data.value - pinValue;
+    const deltaPercentage = Math.round((delta / pinValue) * 100);
```

Comment:
I think we need to watch out for when there are negative metric values.

![image](https://user-images.githubusercontent.com/16869061/219039252-5f25dad7-4a97-4e2c-9c34-0dca6a73693b.png)

This percentage figure here should be positive, right? Going from -3.6 to 1.4 is a positive increase.

Reply:
good catch :)